packages/vue-compat/__tests__/misc.spec.ts

```diff
@@ -275,3 +275,17 @@ test('ATTR_ENUMERATED_COERCION', () => {
     )('contenteditable', 'foo', 'true'),
   ).toHaveBeenWarned()
 })
+
+test('ATTR_ENUMERATED_COERCION: true', () => {
+  const vm = new Vue({
+    compatConfig: { ATTR_ENUMERATED_COERCION: true },
+    template: `<div><div draggable="false">hello</div></div>`,
```

Comment:
I think we should also test boolean `false`. e.g.:
```suggestion
    template: `<div><div draggable="false" :spellcheck="false">hello</div></div>`,
```
Plus the relevant assertions.